sympy/galgebra/ga.py

```diff
@@ -335,14 +335,14 @@ def make_grade2(self, base):  # grade 2 multivector
                 if self.fct:
                     base_lst = str_combinations(base, MV.coords, rank=2, mode='__')
                     fct_lst = fct_sym_array(base_lst, MV.coords)
-                    self.obj = reduce(operator.add, tuple(map(lambda x: x[0] * x[1], zip(fct_lst, MV.blades[2]))))
+                    self.obj = reduce(operator.add, tuple([x[0] * x[1] for x in zip(fct_lst, MV.blades[2])]))
                 else:
                     if MV.coords is not None:
                         base_lst = str_combinations(base, MV.coords, rank=2, mode='__')
                     else:
                         base_lst = str_combinations(base, MV.subscripts, rank=2, mode='__')
                     fct_lst = fct_sym_array(base_lst, None)
-                    self.obj = reduce(operator.add, tuple(map(lambda x: x[0] * x[1], zip(fct_lst, MV.blades[2]))))
+                    self.obj = reduce(operator.add, tuple([x[0] * x[1] for x in zip(fct_lst, MV.blades[2])]))
```

Comment:
> I dimly recall that it's possible write something like x0 * x1 for x0, x1 in fct_list, MV.blades[2].

This is not true, that doesn't work. What you *can* do though is index out into `x0, x1`. So the expression could read:

```
tuple(x0 * x1 for x0, x1 in zip(fct_lst, MV.blades[2]))
```